plutus-tx/plutus-tx.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 2.2
 name: plutus-tx
-version: 0.1.0.0
+version: 0.2.0.0
```

Comment:
We have no versioning policy currently, so there's no point doing this.

Reply:
As per my comment above vis a vis the changelogging, this would immensely help at _least_ one organization currently relying on Plutus. Upgrades are _really_ difficult otherwise at the moment.

Reply:
Literally no Cardano library other than the node has a version. And so long as people are picking things out by git revisions, having a version bound doesn't actually help you much.